src/module/Provider.sol

```diff
@@ -7,13 +7,27 @@ import {MainnetContracts as MC} from "script/Contracts.sol";
 import {IBNBXStakeManagerV2} from "src/interface/external/stader/IBNBXStakeManagerV2.sol";
 import {ISlisBnbStakeManager} from "src/interface/external/lista/ISlisBnbStakeManager.sol";
 import {IAsBnbMinter} from "src/interface/external/astherus/IAsBnbMinter.sol";
+import {IVault} from "src/interface/IVault.sol";
+
+interface IBaseStrategy {
+    function STRATEGY_VERSION() external view returns (string memory);
+}
 
 /*
     The Provider fetches state from other contracts.
 */
 contract Provider is IProvider {
     error UnsupportedAsset(address asset);
 
+    function isETHStrategyVault(address asset) public view returns (bool) {
```

Comment:
please fix this this so it refers to BNB, this is the BNB branch not the ETH branch